stdlib/2/typing.pyi

```diff
@@ -216,7 +223,10 @@ class MutableSet(AbstractSet[_T], Generic[_T]):
     def __ixor__(self, s: AbstractSet[_S]) -> MutableSet[Union[_T, _S]]: ...
     def __isub__(self, s: AbstractSet[Any]) -> MutableSet[_T]: ...
 
-class MappingView(Sized):
+class MappingView:
+    def __len__(self) -> int: ...
+    # Implement Sized (but don't have it as a base class).
+    @abstractmethod
```

Comment:
This has two definitions of `__len__` -- is it supposed to be abstract or not?

Reply:
Whoops. Fixing.